frontend/app/src/components/helper/BlockchainAccountSelector.vue

```diff
@@ -1,13 +1,10 @@
 <script setup lang="ts">
 import { uniqBy } from 'lodash-es';
-import {
-  Blockchain,
-  type BlockchainSelection,
-} from '@rotki/common/lib/blockchain';
+import { Blockchain } from '@rotki/common/lib/blockchain';
 import type { GeneralAccount } from '@rotki/common/lib/account';
 import type { ComputedRef } from 'vue';
 
-type AccountWithChain = GeneralAccount<BlockchainSelection>;
+type AccountWithChain = GeneralAccount;
```

Comment:
guess this is just type re-assignment...

Reply:
yes which I could remove I guess, it made sense with generics before